src/py/flwr/common/serde_test.py

```diff
@@ -320,8 +322,12 @@ def test_message_to_and_from_taskins() -> None:
     taskins.run_id = metadata.run_id
     taskins.task_id = metadata.task_id
     taskins.group_id = metadata.group_id
+    taskins.task.consumer = metadata.node_id
```

Comment:
`consumer` is of type `Node`. I think it should be updated to `taskins.task.consumer.node_id = metadata.node_id`.